crates/egui/src/widgets/drag_value.rs

```diff
@@ -508,8 +508,32 @@ impl<'a> Widget for DragValue<'a> {
 }
 
 fn clamp_to_range(x: f64, range: RangeInclusive<f64>) -> f64 {
-    x.clamp(
-        range.start().min(*range.end()),
-        range.start().max(*range.end()),
-    )
+    match x.total_cmp(range.start()) {
```

Comment:
The previous implementation handles the case of reversed range, i.e. `clamp_to_range(5.0, 10.0..=0.0) == 5.0`. The new versions fails here.

Reply:
Fixed with new commit